mar/last-post.hoon

```diff
@@ -0,0 +1,28 @@
+::
+::::  /hoon/last-post/mar
+  ::
+/?    310
+=,  mimes:html
+=,  html
+|_  url/tape
```

Comment:
its in a renderer/mark because otherwise the only way would be to get the file url through the fora app, which would indiscriminately redirect everyone browsing the fora. I put a description in ren/last-post, I hope thats enough